DuckDuckGoTests/ExternalUrlSchemeTests.swift

```diff
@@ -60,7 +60,7 @@ class ExternalUrlSchemeTests: XCTestCase {
     
     func testThatAboutSchemesAreAllowed() {
         let url = URL(string: "about:blank")!
-        XCTAssertNotEqual(SchemeHandler.schemeType(for: url),
-                          SchemeHandler.SchemeType.navigational)
+        XCTAssertEqual(SchemeHandler.schemeType(for: url),
+                       SchemeHandler.SchemeType.navigational)
```

Comment:
a bug?

Reply:
@mallexxx I think so, [this Equatable check](https://github.com/duckduckgo/iOS/blob/24735a3315ddfabd45c91b203b50fd4c0a0017ce/Core/SchemeHandler.swift#L90-L101) seems wrong to me. Thanks for the fix :) 

What scares me is that this test was built specifically expecting the .navigational == .navigation comparison to be false. I wonder why.